### PR TITLE
Add IsTestProject=true

### PR DESCRIPTION
**Pull request by MarcoRossignoli**

### CODE REVIEW

File: test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
         /// <summary>
         /// The net core versions for which templates are present
         /// </summary>
         private static string[] netCoreVersions = {
             // refer to https://dotnet.microsoft.com/download/dotnet-core
             // for a list of supported dotnet versions and only include the ones
             // that are not end-of-life
-            "3.1",
-            "5.0",
             "6.0",
```
Comment (MarcoRossignoli) [inline test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs:19]: @Evangelink here I'm removing eof versions but I've updated also old template, but we're testing only non eol, what you think? should we:
* test eol
* do not add new prop to the eol
* keep as-is

File: test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs [hunk 2]
```diff
@@ -1,24 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.TestTemplates.Acceptance.Tests
-{
+namespace Microsoft.TestTemplates.Acceptance.Tests {
```
Comment (Evangelink) [inline test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs:4]: The latest VS preview is breaking curly brace position, you will have to manually fix them all

File: test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs (hunk 1 above)
Comment (MarcoRossignoli) [reply test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs:21]: removed eol for now

File: test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs (hunk 2 above)
Comment (MarcoRossignoli) [reply test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs:4]: done

**Pull request merged**